### DISCUSSION TITLE
Follower latency and sync problems

### DISCUSSION THREAD

**Discussion by kubajed** [NightscoutFoundation/xDrip#2222] [Q&A]
Hi,
recently I am encountering problems with my second phone, configured as a follower. My main phone is connected with MiaoMiao, it receives data with no problems nor latency. The second phone scanned QR code from the main phone and has got a source set to xDrip sync follower. This config worked fine for two years, but stopped about two weeks ago. The follower receives data with an increasing delay, eventually leading to lost synchronisation. Please see below screenshots from the follower. The main phone under sync settings has got the option to be a sync source turned on.

<img width="350" src="https://user-images.githubusercontent.com/47507183/178817215-97dcfd42-482d-4171-85b8-f91fe435d0d0.jpg" /> <img width="350" src="https://user-images.githubusercontent.com/47507183/178817241-20b87f7a-a1bf-426f-953b-bc3d6976fda9.jpg" /> <img width="350" src="https://user-images.githubusercontent.com/47507183/178818116-477df3a4-507f-450b-9b95-e2682a4ba80a.jpg" />

**Comment by jamorham** [comment]
Try rebooting one or both phones

**Comment by tolot27** [comment]
Please also update to the latest nightly and verify that the sensor is started.

**Comment by tolot27** [comment]
Is your problem solved?
